cmd/global.go

```diff
@@ -35,18 +34,26 @@ Once you set your global .driveignore you can delete the file you pointed to.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		vPrint := utils.VPrintWrapper(verbose)
 
-		_, currFile, _, _ := runtime.Caller(0)
 		cwd, _ := os.Getwd()
+		configDir, _ := os.UserConfigDir()
 		absPath := filepath.Join(cwd, args[0])
 		content, _ := ioutil.ReadFile(absPath)
 		vPrint("loaded file contents")
 		if shouldAppend {
-			currContent, _ := ioutil.ReadFile(filepath.Join(currFile, "../../.global_driveignore"))
-			ioutil.WriteFile(filepath.Join(currFile, "../../.global_driveignore"), []byte(string(content)+"\n"+string(currContent)), os.ModePerm)
-			vPrint("appended the contents to the .global_driveignore")
+			currContent, _ := ioutil.ReadFile(filepath.Join(configDir, ".global_driveignore"))
+			err := ioutil.WriteFile(filepath.Join(configDir, ".global_driveignore"), []byte(string(content)+"\n"+string(currContent)), os.ModePerm)
+			if err != nil {
+				vPrint(err)
```

Comment:
vPrint is for 'verbose print' therefore the error will never be printed if verbose isnt on. Instead change the function `Run` to `RunE` and return the error.